ksqldb-functional-tests/src/test/resources/query-validation-tests/json_items.json

```diff
@@ -0,0 +1,37 @@
+{
+  "comments": [
+    "Tests covering the use of the JSON_ITEMS function."
+  ],
+  "tests": [
+    {
+      "name": "convert a json array string to an array of json objects",
+      "statements": [
+        "CREATE STREAM test (K STRING KEY, EVENTS STRING) WITH (kafka_topic='test_topic', value_format='JSON');",
+        "CREATE STREAM OUTPUT AS SELECT K, JSON_ITEMS(EVENTS) AS EVENTS FROM test;"
+      ],
+      "inputs": [
+        {"topic": "test_topic", "key": "1", "value": {"events": "[{\"type\": \"A\", \"timestamp\": \"2022-01-27\"}, {\"type\": \"B\", \"timestamp\": \"2022-05-18\"}]"}},
+        {"topic": "test_topic", "key": "1", "value": {"events": "[]"}}
+      ],
+      "outputs": [
+        {"topic": "OUTPUT", "key": "1", "value": {"events": ["{\"type\":\"A\",\"timestamp\":\"2022-01-27\"}", "{\"type\":\"B\",\"timestamp\":\"2022-05-18\"}"]}},
+        {"topic": "OUTPUT", "key": "1", "value": {"events": []}}
+      ]
+    },
+    {
+      "name": "extract timestamps from json objects",
```

Comment:
Thank you for adding an integration test that covers the end-to-end usage of this function!